src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -361,6 +366,23 @@ trait Writer extends AccessLevel {
    */
   val writeKey: String = settings.writeKey.getOrElse(throw MissingCredential("Write key required for Writer"))
 
+  // constants
+  val MinSendInterval: Integer = 60
+  val MaxSendInterval: Integer = 3600
+
+  // initialize and configure our local event store queue
+  val batchSize: Integer = settings.batchSize.getOrElse(500)
+  val batchTimeout: Integer = settings.batchTimeout.getOrElse(5)
```

Comment:
Sorry, it's actually e.g.

```
batch.size = 500
batch.size = ${?KEEN_EVENT_QUEUE_BATCH_SIZE}
```

for the full default-with-env-var-override syntax.